torch/_refs/__init__.py

```diff
@@ -3275,6 +3275,30 @@ def new_empty_strided(
     )
 
 
+@register_decomposition(torch.ops.aten.zeros)
+def zeros(
+    size: ShapeType,
```

Comment:
This should be also a variadic argument, as `torch.zeros(2,3)` is not only a valid construction but it's the main documented overload in the docs.